version.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "v0.2.0"
```

Comment:
Should this be a patch version bump so people can upgrade and things will work, or minor so they don't have to follow us along to go-libipfs?

Reply:
The issue is that `go-libipfs` have newer versions of libs which could break stuff.

For example if I import `example.com@v1.0.0` and go-libipfs imports `example.com@v1.1.0`, updating to `go-ipfs-files@v0.2.1` will update `example.com@v1.0.0` to `example.com@v1.1.0`.

I don't really care about it but that why goreleaser is angry with you when you update go.mod and bump a patch.

Reply:
I updated to 0.3.0